src/index.html

```diff
@@ -7,8 +7,54 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png"
+        alt="logo"
+        class="logo__image">
+      </a>
```

Comment:
apply code styling recommendations from a task checklist. 
Apply for other cases as well
```suggestion
      <a href="#home" class="logo">
        <img 
          src="./images/logo.png"
          alt="logo"
          class="logo__image"
        >
      </a>
```